docs/release-notes/8.11.asciidoc

```diff
@@ -0,0 +1,55 @@
+* In 8.11, rule APIs will only support `investigation_fields` as { field_names: string[] }. Note that, if you've added this field to your rules in 8.10, you don't need to do anything when you import your rules. 
+
+[discrete]
+[[deprecations-8.11.0]]
+==== Deprecations
+* Deprecates the `doc_root.vulnerability.package` and replaces it with the `doc_root.package` ECS package ({pull}164651[#164651]).
+
+[discrete]
+[[features-8.11.0]]
+==== New features
+* Upgrades {elastic-defend} for Windows to capture a new event type: ETW Threat Intelligence (ETW-TI). Renames the Windows events policy `Credential access` category to `API` in the UI (but not in the `.yaml`, maintaining backwards compatibility). Adds two new advanced options: `windows.advanced.events.api_disabled` and
+`windows.advanced.events.api_verbose` ({pull}167549[#167549]).
+* Adds the `Same family` category and tab to the Data Quality dashboard. Fields with mappings in the same family have the same search behavior as the type specified by ECS, but may have different space usage or performance characteristics ({pull}167480[#167480]).
+* Updates the exceptions flyout's `match_any` operator to accept duplicate values that differ in case ({pull}167208[#167208]).
+* Enables the Elastic AI Assistant to answer questions about Elasticsearch Query Language (ES|QL) by allowing it to query, via ELSER, an ES|QL knowledge base. Refer to <<security-assistant, Elastic AI Assistant>> to enable the knowledge base ({pull}167097[#167097]).
+* Enables ES|QL in Timeline (technical preview) ({pull}166764[#166764]).
+* Adds the new ES|QL rule type (technical preview) ({pull}165450[#165450]).
+* Implements Kafka output for {elastic-endpoint} (https://github.com/elastic/endpoint-dev/issues/13192[#13192]).
+* Introduces full support for {elastic-endpoint} on MacOS (https://github.com/elastic/endpoint-dev/issues/13058[#13058]).
+
+
+[discrete]
+[[enhancements-8.11.0]]
+==== Enhancements
+* Adds a new Generative AI connector, Amazon Bedrock, for use with Elastic AI Assistant ({pull}166662[#166662]).
+* Adds the `id`, `severity`, and `status` fields to the Webhook - Case Management connector ({pull}166295[#166295]).
+* Updates the order of items on {kib}'s left-side navigation menu to match the order in {elastic-sec}'s left-side navigation menu ({pull}164268[#164268]).
+* Updates the Endpoint policy UI (**Manage -> Policies**) to include a `Protection updates` tab, a new column called `Deployed version`, and a banner that highlights outdated policies ({pull}165256[#165256], {pull}162719[#162719]).
+* Adds tooltips to overview section titles in the alert details flyout ({pull}166737[#166737]).
+The `.lists` and `.items` indices were moved to data streams ({pull}162508[#162508]).
```

Comment:
```suggestion
* Updates the `.lists` and `.items` indices to data streams ({pull}162508[#162508]).
* Renames the Generative AI connector to OpenAI, since Generative AI is now a category of connectors that include OpenAI and Amazon Bedrock ({pull}167677[#167677]).
```